OsmTileLoader.cpp

```diff
@@ -4,6 +4,7 @@
 #include <curl/curl.h>
 #include <future>
 #include <iostream>
+#include <memory.h>
```

Comment:
This likely should be just `#include <memory>`, as the C++ headers do not normally include an extension.

Reply:
Sure, thanks, fixed on the master branch.